package.json

```diff
@@ -203,12 +203,6 @@
     "webpack-hot-middleware": "^2.18.2",
     "xml2json": "^0.11.0"
   },
-  "pre-commit": [
-    "lint-staged"
```

Comment:
I agree with this change... linting should be checked by the CI after push, but _before_ merging.